src/data/services/EnterpriseAccessApiService.js

```diff
@@ -159,6 +159,17 @@ class EnterpriseAccessApiService {
     return EnterpriseAccessApiService.apiClient().get(url);
   }
 
+  /**
+   * Cancel content assignments for a specific AssignmentConfiguration.
+   */
+  static cancelContentAssignments(assignmentConfigurationUUID, uuids) {
+    const options = {
+      assignment_uuids: Array.isArray(uuids) ? uuids : [uuids],
```

Comment:
similar feedback here around consistency of data types passed to `uuids`. For example, could the caller of `cancelContentAssignments` ensure an array is always passed?